ccan/ccan/htable/htable.h

```diff
@@ -75,6 +75,15 @@ bool htable_init_sized(struct htable *ht,
 		       size_t (*rehash)(const void *elem, void *priv),
 		       void *priv, size_t size);
 
+/**
```

Comment:
Just to be clear, the `htable` changes are not part of the fix, only the `pipecmd` changes are?  Basically `htable` just happened to get pulled in due to the CCAN update?

Reply:
Yep!  They're also a useful cleanup (we access .raw.elems in some places) but I'll do that as part of another tree I have waiting for 0.7.3...